pkg/spec/backup.go

```diff
@@ -73,7 +73,24 @@ type PVSource struct {
 	VolumeSizeInMB int `json:"volumeSizeInMB"`
 }
 
+// TODO: support per cluster S3 Source configuration.
 type S3Source struct {
+	// The name of the AWS S3 bucket to store backups in.
+	//
+	// S3Bucket overwrites the default etcd operator wide bucket.
+	S3Bucket string `json:"s3Bucket,omitempty"`
+
+	// The name of the secret object that stores the AWS credential file.
+	// The file name must be 'credentials'.
+	//
+	// AWSCredentialSecret overwrites the default etcd operator wide AWS credential.
+	AWSCredentialSecret string `json:"awsCredentialSecret,omitempty"`
```

Comment:
sounds good to me. I also feel so. 

are we going to deprecate the old behavior in etcd operator?